### ISSUE TITLE
rh_certificates.rb fails on nodes without subscription manager data

### ISSUE THREAD

**Issue by csschwe** [theforeman/puppet-foreman_scap_client#20]
rh_certificates.rb seems to fail if `data = Facter::Util::Resolution.exec('subscription-manager config')` returns an empty string.

```
Error: Could not retrieve local facts: undefined method '+' for nil:NilClass
Error: Failed to apply catalog: Could not retrieve local facts: undefined method '+' for nil:NilClass
```

https://github.com/theforeman/puppet-foreman_scap_client/blob/master/lib/facter/rh_certificates.rb#L7

Testing data as such seems to fix the problem
`return nil if data.nil? || data == ''`

**Comment by shlomizadok** [comment]
Create a pull request?
also, maybe change to `return nil if data.nil? || data.blank?`
Thanks

**Comment by shlomizadok** [comment]
fixed with #21 - closing
